cfme/tests/infrastructure/test_resources.py

```diff
@@ -0,0 +1,25 @@
+from utils import testgen
+import pytest
+
+pytestmark = [pytest.mark.meta(server_roles='+smartproxy +smartstate')]
+
+
+pytest_generate_tests = testgen.generate(testgen.provider_by_type,
+                                         ['openstack-infra'],
+                                         scope='module')
+
+
+@pytest.mark.usefixtures("setup_provider_modscope")
```

Comment:
And mfalesni's comment about the .value.number fomr before still needs addressing